pkg/app/app.go

```diff
@@ -44,7 +47,11 @@ func initConfiguration(config configuration.Configuration, apiClient api.ApiClie
 
 	config.AddDefaultValue(configuration.WEB_APP_URL, func(existingValue any) any {
 		canonicalApiUrl := config.GetString(configuration.API_URL)
-		appUrl, _ := api.DeriveAppUrl(canonicalApiUrl)
+		appUrl, err := api.DeriveAppUrl(canonicalApiUrl)
+		if err != nil {
+			logger.Println("Failed to determine default value for \"WEB_APP_URL\":", err)
```

Comment:
Same here - should we exit with RC=2 as this must not happen?

Reply:
I'm careful with exiting since it might be a too aggressive approach and might break things unexpectedly. I think step one is logging to increase transparency.